app/components/add-layer/component.js

```diff
@@ -206,19 +207,30 @@ export default Ember.Component.extend({
                             "right"
                             ],
                             "validation": null
+                        },
+                        {
+                         "type": "settings",
+                         "items": [
+                         {
+                            "type": "checkbox",
+                            "label": "Stick to top of page on scroll",
+                            "value": "StickToTop",
```

Comment:
Let's rethink whether we should make this an option. In any case the default should be true.